CHANGELOG.md

```diff
@@ -1,3 +1,153 @@
+# 103.2.0
+
+Replace this text with a summarized description of this release's contents.
+
+## Breaking changes
+
+Replace this explanations for how to resolve the breaking changes.
+## New deprecations
+
+Replace this text with links to deprecation guides.
+## New features
+
+AppBar added an API for changing the stack view offset. 
+Flexible header added an API for showing the hairline and its color.
+Availability macro for internal use.
+AlertControllers added an API for titleIconView.
+Bottom drawer added an API for sticky status bars and adjustment when content size changes.
+Slider added an API for allowAnimatedValueChanges and trackEndsAreRounded or trackEndsAreInset.
+
+## API changes
```

Comment:
This can probably be deleted

Reply:
JK, components/Buttons/src/ButtonThemer/MDCFloatingActionButtonThemer.h was deleted